@navikt/core/react/src/util/composition-warning/CompositionWarning.tests.stories.tsx

```diff
@@ -0,0 +1,120 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { expect, within } from "@storybook/test";
+import React, { PropsWithChildren, useLayoutEffect, useState } from "react";
+import { CompositionWarning } from ".";
+
+export default {
+  title: "Utilities/CompositionWarning/Tests",
+  parameters: {
+    chromatic: { disable: true },
+  },
+} satisfies Meta;
+
+type Story = StoryObj;
+
+function ConsoleWarnCapture({ children }: PropsWithChildren) {
+  const [count, setCount] = useState(0);
+  const [lastMessage, setLastMessage] = useState<string>("");
+
+  useLayoutEffect(() => {
+    const original = console.warn;
+    const override: typeof console.warn = (...args) => {
+      setCount((c) => c + 1);
+      if (args.length > 0) {
+        setLastMessage(String(args[0]));
+      }
+      return original.apply(console, args as unknown as [any]);
```

Comment:
Seems to work without the type casting
```suggestion
      return original.apply(console, args);
```